analysis_options.yaml

```diff
@@ -10,45 +10,65 @@ analyzer:
     - "test/goldens/generated_build_script.dart"
 linter:
   rules:
-    # Errors
-    - avoid_empty_else
-    - await_only_futures
-    - comment_references
-    - control_flow_in_finally
-    - empty_statements
-    - hash_and_equals
-    - iterable_contains_unrelated_type
-    - no_duplicate_case_values
-    - test_types_in_equals
-    - throw_in_finally
-    - unawaited_futures
-    - unnecessary_statements
-    - unrelated_type_equality_checks
-    - valid_regexps
-
-    # Style
     - annotate_overrides
+    - avoid_empty_else
     - avoid_function_literals_in_foreach_calls
     - avoid_init_to_null
+    - avoid_null_checks_in_equality_operators
     - avoid_return_types_on_setters
     - avoid_returning_null
     - avoid_unused_constructor_parameters
+    - await_only_futures
     - camel_case_types
+    #- cancel_subscriptions
+    #- cascade_invocations
+    - comment_references
+    #- constant_identifier_names
+    - control_flow_in_finally
     - directives_ordering
     - empty_catches
     - empty_constructor_bodies
+    - empty_statements
+    - hash_and_equals
+    #- implementation_imports
+    - invariant_booleans
+    - iterable_contains_unrelated_type
     - library_names
     - library_prefixes
+    - list_remove_unrelated_type
+    - no_adjacent_strings_in_list
+    - no_duplicate_case_values
     - non_constant_identifier_names
+    - omit_local_variable_types
+    #- only_throw_errors
+    - overridden_fields
+    - package_api_docs
```

Comment:
Drop this. It's encourages worthless comments.

Reply:
Fixed